Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -1787,6 +2090,39 @@ def test_module(client: MsClient):
 ''' EXECUTION CODE '''
 
 
+def update_indicator_command(client: MsClient, args: dict) -> Tuple[str, dict]:
```

Comment:
these shouldnt be in the EXEC CODE section, please place them above the test_module func and below the other funcs you added